### ISSUE TITLE
Allow having an access token and refresh token in the same request

### ISSUE THREAD

**Issue by vimalloc** [vimalloc/flask-jwt-extended#298]
Being able to pass both token in as part of the same request would make logging out when using access and refresh tokens a much nicer experience. Something like this:

```python
@jwt_required(optional=True)
@jwt_required(optional=True, refresh=True)
@app.route('/logout', methods=['DELETE'])
def logout():
    access_jti = get_jti(get_access_token())
    refresh_jti = get_jti(get_refresh_token())
    # blacklist both access and refresh tokens
```

**Comment by vimalloc** [comment]
The biggest hurdle with this will be making `get_raw_jwt`, `current_user`, et al work when there is multiple tokens in the request. Keeping these working as they are now will be the best user experience, because in 99% of endpoints there should only be one JWT present in the request.

I think the best way to handle this would be something along the lines of adding a `get_raw_jwts` method that returns a list of current valid JWTs in the request, and then have the other methods like `get_raw_jwt` or `current_user` either raise an error if there is more then one token in the request, or just always use the first/last jwt in the request.

**Comment by vimalloc** [comment]
I have done a lot of thinking about this, and just could not come up with a transparent way that I like at all. Instead, I think what I am going to do is create an option for the access tokens and refresh tokens to come in different headers, then you can handle them both in the same request explictly with something like this:

```python
@app.route('/logout', methods=['DELETE'])
def logout():
    verify_jwt_in_request(refresh=False)
    add_jti_to_blacklist(get_jwt()['jti'])

    verify_jwt_in_request(refresh=True)
    add_jti_to_blacklist(get_jwt()['jti'])

    return jsonify({'logout': True})
```

**Comment by vimalloc** [comment]
Ok here is my last idea. Add a `get_access_jwt`and `get_refresh_jwt` functions, then make `get_jwt` raise a helpful error message saying to use one of these new functions if both access and refresh tokens exist in this request, otherwise just return whatever jwt happens to be present. This should be much more discoverable and easier to work with then my other approaches.

**Comment by vimalloc** [comment]
I'm actually going to remove this for the 4.0.0 release. After going back and forth on a bunch of different versions, I'm just not sure how, or even if, I want to do this. There are more complications then I had realized going into this, and I don't want to hold up the 4.0.0 version on this. 

Another interesting point is I was reading the spec closer, and authorization headers should not support multiple tokens in the first place. This is something that we do currently support, and I'm hesitant to remove that functionality in case people are relying on it, but it does for sure make me not want to double down on supporting it. 

Closing this issue for now.
